scripts/generate_grpc_stubs.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+STUBS_GENERATOR="bentoml/stubs-generator"
+
+cd "$GIT_ROOT" || exit 1
+
+main() {
+	local VERSION="${1:-v1alpha1}"
+	# Use inline heredoc for even faster build
+	# Keeping image as cache should be fine since we only want to generate the stubs.
+	if [[ $(docker images --filter=reference="$STUBS_GENERATOR" -q) == "" ]] || test "$(git diff --name-only --diff-filter=d -- "$0")"; then
+		docker buildx build --platform=linux/amd64 -t "$STUBS_GENERATOR" --load -f- . <<EOF
+# syntax=docker/dockerfile:1.4-labs
+
+FROM --platform=linux/amd64 python:3-slim-bullseye
+
+ENV DEBIAN_FRONTEND noninteractive
+
+WORKDIR /workspace
+
+COPY <<-EOT /workspace/requirements.txt
+    # Restrict maximum version due to breaking protobuf 4.21.0 changes
+    # (see https://github.com/protocolbuffers/protobuf/issues/10051)
+    protobuf==3.19.4
```

Comment:
Why are we locking to such a new version?

Reply:
grpcio-tools==1.41.0 requires protobuf>=3.15.0, and 3.10 support are provides from protobuf>=3.19.0

Reply:
Sounds good. Should we add a quick comment here explaining the reasoning?

Reply:
will do.